src/SFA.DAS.EmployerAccounts.Web/Orchestrators/UserAornLockOrchestrator.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System.Threading.Tasks;
+using AutoMapper;
+using MediatR;
+using SFA.DAS.EmployerAccounts.Models.UserProfile;
+using SFA.DAS.EmployerAccounts.Queries.GetUserAornLock;
+using SFA.DAS.EmployerAccounts.Queries.UpdateUserAornLock;
+using SFA.DAS.EmployerAccounts.Web.ViewModels;
+
+namespace SFA.DAS.EmployerAccounts.Web.Orchestrators
+{
+    public class UserAornLockOrchestrator
```

Comment:
I don't think this is an orchestrator.  The (truly awful) pattern has a 1-2-1 relationship between controllers and orchestrators.